### PR TITLE
Added isDisplayed.

### PR DESCRIPTION
**Pull request by Saurabh-LT**

### CODE REVIEW

File: lambdatest.py [hunk 1]
```diff
@@ -62,6 +59,7 @@ def test_demo_site(self):
 
         # Searching for the results
         search = driver.find_element(By.XPATH, "//*[@id='search']")
+        print("Search button is displayed " + str(search.is_displayed()))
```
Comment (sururocks) [inline lambdatest.py:62]: does it gonna fail?
it should fail if that command is not working.
add assert.

File: lambdatest.py [hunk 2]
```diff
@@ -22,7 +21,7 @@ def setUp(self):
                 "console": 'true',  # Enable or disable console logs
                 "network": 'true'  # Enable or disable network logs
             },
-            "browserName": "Chrome",
+            "browserName": "firefox",
```
Comment (sururocks) [inline lambdatest.py:24]: you supposed to add code for selenium 4

**Review by sururocks — CHANGES_REQUESTED** [review]
change requested

File: lambdatest.py (hunk 2 above)
Comment (Saurabh-LT) [reply lambdatest.py:24]: It's already added

Comment (Saurabh-LT) [reply lambdatest.py:24]: line number 20

**Pull request closed without merging**